modules/nlu/src/backend/legacy-election.ts

```diff
@@ -0,0 +1,169 @@
+import * as sdk from 'botpress/sdk'
+import * as math from './tools/math'
+import _ from 'lodash'
+
+const OOS_AS_NONE_TRESH = 0.3
+const LOW_INTENT_CONFIDENCE_TRESH = 0.4
+const NONE_INTENT = 'none' // should extract constant in comon place
+
+type EventUnderstanding = sdk.IO.EventUnderstanding
```

Comment:
Just so you know you don't have to import * as sdk everywhere,

you ca also `import { IO } from 'botpress/sdk'` and maybe import only types from specific namespaces